src/sql/workbench/electron-browser/modelComponents/queryTextEditor.ts

```diff
@@ -132,8 +132,8 @@ export class QueryTextEditor extends BaseTextEditor {
 
 	public setHeightToScrollHeight(configChanged?: boolean): void {
 		let editorWidget = this.getControl() as ICodeEditor;
-		if (!this._config) {
-			this._config = new Configuration(true, undefined, editorWidget.getDomNode(), this.accessibilityService);
+		this._config = new Configuration(true, undefined, editorWidget.getDomNode(), this.accessibilityService);
```

Comment:
Why always recreate this?

Reply:
Ugh it didn't respond to changes in the node in the DOM if I didn't always recreate it.

I tried doing a number of things, including saving editorWidget to the class' scope, and this was the only way that I could find. Sigh.